salt/pkgrepo/ubuntu/init.sls

```diff
@@ -1,3 +1,16 @@
+legacy-saltstack-pkgrepo:
```

Comment:
@basepi Would it be preferable to make this a new (noun?) similar to the `foo-new` or `foo-nq` stuff I've noticed in other `saltstack-formulas` repos?